t/Net/DRI/Protocol/EPP/Core.t

```diff
@@ -552,7 +552,7 @@ $R2=$E1.'<greeting><svID>Example EPP server epp.example.com</svID><svDate>2000-0
 $rc=$dri->process('session','noop',[]);
 $R2='';
 $rc=$dri->process('session','login',['ClientX','foo-BAR2',{only_local_extensions => 1}]);
```

Comment:
I feel like `only_local_extensions` has been upset by the way some extensions run setup (see sub setup in VeriSign::Sync). Its not critical, but it required that these extensions are added to the test.